lib/AliasFieldPlugin.js

```diff
@@ -41,11 +40,11 @@ AliasFieldPlugin.prototype.apply = function(resolver) {
 			path: request.descriptionFileRoot,
 			request: data
 		});
-		resolver.doResolve(target, obj, "aliased from description file " + request.descriptionFilePath + " with mapping '" + innerRequest + "' to '" + data + "'", createInnerCallback(function(err, result) {
+		resolver.doResolveWithErrorLogging(target, obj, "aliased from description file " + request.descriptionFilePath + " with mapping '" + innerRequest + "' to '" + data + "'", function(err, result) {
```

Comment:
`doResolve` skips the "run twice", `resolve` doesn't unless I'm misunderstanding?